feature-libs/organization/administration/core/services/organization-page-meta.resolver.spec.ts

```diff
@@ -152,18 +139,38 @@ describe('OrganizationPageMetaResolver', () => {
     });
   });
 
-  describe('resolveRobots', () => {
-    it('should resolve title from the BasePageMetaResolver', async () => {
-      spyOn(basePageMetaResolver, 'resolveRobots').and.returnValue(
-        of([PageRobotsMeta.FOLLOW, PageRobotsMeta.INDEX])
-      );
-      let result;
-      resolver
-        .resolveRobots()
-        .subscribe((robots) => (result = robots))
-        .unsubscribe();
-      expect(result).toContain(PageRobotsMeta.FOLLOW);
-      expect(result).toContain(PageRobotsMeta.INDEX);
-    });
+  it(`should resolve 'Page description' for resolveDescription()`, () => {
+    let result: string | undefined;
+
+    spyOn(basePageMetaResolver, 'resolveDescription').and.returnValue(
```

Comment:
These tests should now use only ContentPageMetaResolver

Reply:
yes fixed it this morning to pass tests